api/src/routes.ts

```diff
@@ -6,4 +6,13 @@ export const router = Router();
 
 router.all('/health', GeneralController.health);
 router.all('/events/click-event', EventController.clickEvent);
+router.get('/events/click-events', EventController.paginatedClickEvents)
+
+router.all('/events/input-event', EventController.inputEvent);
+router.get('/events/input-events', EventController.paginatedInputEvents)
+
+
+router.all('/events/visit-event', EventController.visitEvent);
+router.get('/events/visit-events', EventController.paginatedVisitEvents)
```

Comment:
don't need a seperate router.get if you use a unified object for the route:

i.e., see https://github.com/GTBitsOfGood/bog-analytics/pull/21/files#r1493344769